index.js

```diff
@@ -59,13 +60,22 @@ module.exports = function(source) {
     }
   });
 
+  manualImports = manualImports.map(function (it) { return utils.lookup(it, options.paths, '', true); });
```

Comment:
Ah something just clicked for me. We need to resolve the imports before resolving there dependencies since we need to know the full path to look at. I should have picked up on this faster. Sorry.

We should use the resolvers in PathCache to asynchronously do that to get webpack resolutions too. Need little code like https://github.com/shama/stylus-loader/blob/master/lib/pathcache.js#L135-L138 with some normalization probably to an array. Probably as a method in `pathcache.js` that we'll use here.